src/components/Modal/tests/Modal.test.tsx

```diff
@@ -403,16 +402,15 @@ describe('<Modal>', () => {
     });
 
     it('focuses the activator when the activator is an element on close', () => {
-      const focusSpy = jest.spyOn(focusUtils, 'focusFirstFocusableNode');
-
+      const id = 'activator-id';
       const modal = mountWithApp(
-        <Modal onClose={noop} open activator={<Button />} />,
+        <Modal onClose={noop} open activator={<Button id={id} />} />,
       );
 
       modal.find(Dialog)!.trigger('onExited');
+      const activator = modal.find('button', {id})!.domNode;
 
-      expect(document.activeElement).toBe(modal.find('button')!.domNode);
-      expect(focusSpy).toHaveBeenCalledTimes(3);
```

Comment:
@LauraAubin I saw you authored the test. Was there a reason we had to test the implementation (`focusFirstFocusableNode`)? It looks like it was originally `toHaveBeenCalledTimes(2);` & was changed in v5 to be `toHaveBeenCalledTimes(3);`. Rather than altering it again, I feel like checking that the activator is sufficient. What do you think?